docs/install.markdown

```diff
@@ -25,6 +25,10 @@ not covered by the list above.
 
 ## Build/Install
 
+Currently Watchman can be installed either through: 
+1. Homebrew for OS X
+2. Source for other systems
```

Comment:
This implies that we only support building via homebrew on OS X, which isn't the case.  Do we need to add this exact text?  Is there something you found unclear?